src/app/shared/modules/routing.module.ts

```diff
@@ -0,0 +1,16 @@
+import { NgModule } from '@angular/core';
```

Comment:
this file is supposed to live within moduleName-routing.module.ts
as you have only 1 module, this should be within app-routing.module.ts
shared (module) is used when we have multiple modules and you need to connect some general things (functions, components) in several modules
again, as you have 1 module -shared is redundant 
all the folders you currently have in shared can be moved to app folder, material can be separate module
we don't need folder modules as angular app is built with modules as separate details and they shouldn't be placed in 1 folder (as one-module components)
also, shared should be super general module and we should avoid store within it